doc/releases/reef.rst

```diff
@@ -6,6 +6,432 @@ Reef is the 18th stable release of Ceph. It is named after the reef squid
 (Sepioteuthis).
 
 
+v18.2.4 Reef
+============
+
+This is the fourth backport release in the Reef series. We recommend that all users update to this release.
```

Comment:
Officially, it's the third release but I guess it depends on how you look at it ;)